### PR TITLE
Add a basic game

### PR DESCRIPTION
**Pull request by byronsharman**
This is a very simple game that should be enough to start the networking part of the project.

### CODE REVIEW

File: src/game/game.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+        # the 'or 1' causes division by 1 if the magnitude is 0 to avoid zero division errors
+        #
+        # Note: if it is possible to standardize thrusts on a scale of 0 to 1,
+        #       this could be optimized by dividing by sqrt(2) or not at all
+        thrust /= np.sqrt((thrust**2).sum()) or 1
+        # The previous line normalized the thrust's magnitude to 1. Now we
+        # change that magnitude to Character.THRUST simply by multiplication.
+        thrust *= Character.THRUST
+
+        self.velocity += thrust
```
Comment (angus-lherrou) [inline src/game/game.py:70]: Do we want to add a decay to velocity to simulate friction/air resistance, or do we want to think of this game as being in space or something?

This is a design decision we should make--I'm fine either way but it's just something I thought of.

**Review by angus-lherrou — COMMENTED** [review]
Looks good to me modulo the flake8 complaints.

File: src/game/game.py (hunk 1 above)
Comment (byronsharman) [reply src/game/game.py:70]: That's a great question. I propose that we hold a discussion about what sort of theme the game will have (spaceships? animals? airplanes? gardening?) and use that information to decide whether to add any sort of velocity decay.

Comment (angus-lherrou) [reply src/game/game.py:70]: Sounds good. I'll mark this as resolved and we can discuss that in #4.

**Pull request merged**